modules/logging-loki-cli-install.adoc

```diff
@@ -18,28 +16,242 @@ To install and configure logging on your {product-title} cluster, additional Ope
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `namespace` object for the {clo}:
++
+.Example `namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+name: openshift-logging # <1>
+annotations:
+    openshift.io/node-selector: ""
+labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> The Red{nbsp}Hat OpenShift Logging Operator is only deployable to the `openshift-logging` namespace.
+<2> A string value that specifies the label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+. Apply the `namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create an `OperatorGroup` object
++
+.Example `OperatorGroup` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+<1> You must specify the `openshift-logging` namespace.
+
+. Apply the `OperatorGroup` object by running the following command:
 +
 [source,terminal]
 ----
 $ oc apply -f <filename>.yaml
 ----
+
+. Create a `Subscription` object:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable # <2>
+  name: cluster-logging
+  source: redhat-operators # <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+
+. Create a `LokiStack` CR:
++
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 # <4>
+      type: s3 # <5>
+      credentialMode: # <6>
+  storageClassName: <storage_class_name> # <7>
+  tenants:
+    mode: openshift-logging # <8>
+----
+<1> Use the name `logging-loki`.
+<2> You must specify the `openshift-logging` namespace.
+<3> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<4> Specify the name of your log store secret.
+<5> Specify the corresponding storage type.
```

Comment:
1. It's not explained what are the types supported at this point, then, an user won't know what types to choose at this point. The supported are https://docs.openshift.com/container-platform/4.17/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage_installing-log-storage
2. Needed to change "storage type" by "object storage type"